README.md

```diff
@@ -42,6 +42,20 @@ You can use the built-in sbt shell in IntelliJ without any problems. However, th
 
 Additionally, you might have to run `sbt buildInfo` if you encounter an error like `object BuildInfo is not a member of package com.microsoft.hyperspace` for the first build.
 
+To debug with the sbt shell in IntelliJ, run `set Test/fork := false` in the sbt shell so that the debugger attached to the sbt shell can work. Without this, breakpoints won't work.
```

Comment:
It would be good to add a new md file for "Using IntelliJ" as this is for the front page. WDYT?

Reply:
I think that's better. Thanks!